openscap_daemon/evaluation_spec.py

```diff
@@ -349,6 +349,16 @@ def get_cpe_ids(self, config):
 
         return cpe_ids
 
+    def select_profile_by_suffix(self, ssg_sds, profile_suffix):
```

Comment:
This shouldn't take `ssg_sds`. Instead it should use whatever content is already set in the `EvaluationSpec`. If there is no content set yet it should throw an exception.